scripts/installDepsScripts/installDotnetSDK8.sh

```diff
@@ -26,14 +26,14 @@ if [ "$source" != "https://packages.microsoft.com/ubuntu/$repo_version/prod" ];
     rm packages-microsoft-prod.deb
 
     # Remove the dotnet-sdk installation that doesn't come from the Microsoft source
-    sudo apt-get remove $package -y
+    sudo apt-get remove --purge $package -y
 
     sudo apt-get autoremove -y
 
 fi
 
 # Update packages
-sudo apt update -y
+sudo apt-get update -y
```

Comment:
for the .NET 8 is not needed? Why?

Reply:
Actually not needing it is the [expected behavior](https://learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#register-the-microsoft-package-repository) for the dotnet-sdk package installed through Microsoft repository. However, at least on my Ubuntu 22.04 and on the test machine with Windows and WSL (Ubuntu 22.04) that is on the office, it worked as expected just for .NET 8. For .NET 6 and 7 it was not giving priority for all the dotnet packages coming from the Microsoft repository, so to fix this it was necessary to [enforce this priority by manually adding it to the /etc/apt/preferences file](https://learn.microsoft.com/en-us/dotnet/core/install/linux-package-mixup?pivots=os-linux-ubuntu#i-need-a-version-of-net-that-isnt-provided-by-my-linux-distribution)

Reply:
@andreriesco IMHO we should play in the safe here and always use the Microsoft repo. I kind of do not trust the Ubuntu one, this can change like it changed for the other versions.

Reply:
Actually I don't think they intentionally prioritized the Ubuntu packages for .NET 6 and .NET 7, I think they messed up something in the package that it doesn't prioritized it's dependencies from the Microsoft repo
```
Get:1 http://br.archive.ubuntu.com/ubuntu jammy-updates/main amd64 dotnet-targeting-pack-6.0 amd64 6.0.128-0ubuntu1~22.04.2 [2.248 kB]
Get:2 http://br.archive.ubuntu.com/ubuntu jammy-updates/main amd64 dotnet-hostfxr-6.0 amd64 6.0.128-0ubuntu1~22.04.2 [155 kB]
Get:3 http://br.archive.ubuntu.com/ubuntu jammy/main amd64 liblttng-ust-common1 amd64 2.13.1-1ubuntu1 [27,1 kB]
Get:4 http://br.archive.ubuntu.com/ubuntu jammy/main amd64 liblttng-ust-ctl5 amd64 2.13.1-1ubuntu1 [77,9 kB]
Get:5 http://br.archive.ubuntu.com/ubuntu jammy/main amd64 liblttng-ust1 amd64 2.13.1-1ubuntu1 [190 kB]
Get:6 http://br.archive.ubuntu.com/ubuntu jammy-updates/main amd64 dotnet-runtime-6.0 amd64 6.0.128-0ubuntu1~22.04.2 [23,6 MB]
Get:7 http://br.archive.ubuntu.com/ubuntu jammy-updates/main amd64 aspnetcore-runtime-6.0 amd64 6.0.128-0ubuntu1~22.04.2 [7.314 kB]
Get:8 http://br.archive.ubuntu.com/ubuntu jammy-updates/main amd64 dotnet-apphost-pack-6.0 amd64 6.0.128-0ubuntu1~22.04.2 [3.704 kB]
Get:9 http://br.archive.ubuntu.com/ubuntu jammy-updates/main amd64 aspnetcore-targeting-pack-6.0 amd64 6.0.128-0ubuntu1~22.04.2 [1.457 kB]
Get:10 https://packages.microsoft.com/ubuntu/22.04/prod jammy/main amd64 dotnet-sdk-6.0 amd64 6.0.421-1 [86,8 MB]
```

Reply:
But for .NET 8 the package correctly prioritizes also the dependencies from the Microsoft repo
```
Get:1 https://packages.microsoft.com/ubuntu/22.04/prod jammy/main amd64 dotnet-hostfxr-8.0 amd64 8.0.4-1 [108 kB]
Get:2 https://packages.microsoft.com/ubuntu/22.04/prod jammy/main amd64 dotnet-runtime-deps-8.0 amd64 8.0.4-1 [2.886 B]
Get:3 https://packages.microsoft.com/ubuntu/22.04/prod jammy/main amd64 dotnet-runtime-8.0 amd64 8.0.4-1 [23,0 MB]
Get:4 https://packages.microsoft.com/ubuntu/22.04/prod jammy/main amd64 aspnetcore-runtime-8.0 amd64 8.0.4-1 [7.672 kB]
Get:5 https://packages.microsoft.com/ubuntu/22.04/prod jammy/main amd64 dotnet-targeting-pack-8.0 amd64 8.0.4-1 [2.802 kB]
Get:6 https://packages.microsoft.com/ubuntu/22.04/prod jammy/main amd64 aspnetcore-targeting-pack-8.0 amd64 8.0.4-1 [1.901 kB]
Get:7 https://packages.microsoft.com/ubuntu/22.04/prod jammy/main amd64 dotnet-apphost-pack-8.0 amd64 8.0.4-1 [3.517 kB]
Get:8 https://packages.microsoft.com/ubuntu/22.04/prod jammy/main amd64 dotnet-sdk-8.0 amd64 8.0.204-1 [110 MB]
```

Reply:
So as this package works correctly I was trying to avoid making changes the user's `/etc/apt/preferences` file. Although I don't see this changes leading to any problem unless the user has any set there any preference for `dotnet* aspnet* netstandard*` packages, the idea of not adding this was to making changes to the user configurations unless it's necessary.

Reply:
but also we install powerhell that comes from the Microsoft repo, so, I'm do not see a problem here to set it to come all from the Microsoft repo.

Reply:
But for powershell the installation didn't make changes the `/etc/apt/preferences` file also. If the package installation is working correctly this prioritization of Microsoft packages works by default. But I mean, I don't see any drawbacks of adding this configuration to the `/etc/apt/preferences`, just more of the idea of avoiding making changes to the user configurations, so I will add it also to .NET 8 to make sure it always installs packages coming from the Microsoft repo.